cloudigrade/account/serializers.py

```diff
@@ -142,11 +142,12 @@ def add_openshift_tag(self, session, ami_id, ami_region, image):
 
         """
         ec2_image = aws.get_ami(session, ami_id, ami_region)
-        has_openshift = 'cloudigrade-ocp-present' in [
-            tags['Key'] for tags in ec2_image.tags]
-        if has_openshift:
-            image.tags.add(ImageTag.objects.filter(
-                description='openshift').first())
+        if ec2_image.tags is not None:
```

Comment:
So it is either `None` or is a dictionary with the key `'Key'` all the time?

Reply:
Yep. Unless amazon changes their APi.